source/plugins/go/src/utils_linux.go

```diff
@@ -15,3 +19,14 @@ func EnsureGenevaOr3PNamedPipeExists(namedPipeConnection *net.Conn, datatype str
 	Log("Error::EnsureGenevaOr3PNamedPipeExists not implemented for Linux")
 	return false
 }
+
+func isProcessRunning(processName string) (string, error) {
```

Comment:
did you verify this code works? linux process name will not have .exe, so you would need to trim that if the processName has .exe in it

Reply:
yes verified works please check screenshot, also in the code line 303. it checked the os type before passing in telegraf.exe: if osType != "" && strings.EqualFold(osType, "windows")